src/BinSkim.Rules/PERules/BA4001.ReportPortableExecutableCompilerData.cs

```diff
@@ -15,16 +15,14 @@
 namespace Microsoft.CodeAnalysis.IL.Rules
 {
     [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor)), Export(typeof(IOptionsProvider))]
-    public class ReportPECompilerData : WindowsBinaryAndPdbSkimmerBase, IOptionsProvider
+    public class ReportPortableExecutableCompilerData : PEBinarySkimmerBase, IOptionsProvider
```

Comment:
>ReportPortableExecutableCompilerData [](http://example.com/codeflow?start=17&length=36)

I saw that  the file name didn't match with the class name. So, I did this renaming.

This will update all SARIF files, showing a renaming.